tina/collections/rules.ts

```diff
@@ -51,27 +75,40 @@ export const Rules: Collection = {
     {
       type: 'datetime',
       name: 'created',
+      description: 'If you see this field, contact a dev immediately 😳 (should be a hidden field generated in the background).',
       label: 'Created',
+      ui: {
+        component: 'hidden',
+
+      }
     },
     {
       type: 'string',
       name: 'archivedreason',
       label: 'Archived Reason',
+      description: 'If this rule has been archived, summarise why here.',
     },
     {
       type: 'string',
       name: 'guid',
       label: 'Guid',
+      description: 'If you see this field, contact a dev immediately 😳 (should be a hidden field generated in the background).',
+      ui: {
+        component: 'hidden',
+
+      }
     },
     {
       type: 'string',
       name: 'seoDescription',
       label: 'SEO Description',
+      description: 'A summary of the page content, used for SEO purposes. This can be generated automatically with AI.',
     },
     {
       type: 'rich-text',
       name: 'body',
       label: 'Body',
+      description: 'Rule content. This is where the magic happens.',
```

Comment:
I would expect a link to the wiki (I know its not relevant at the moment)

Reply:
The wiki? 🤔

Reply:
A new section here
https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Edit-Rules#how-to-edit-rules-using-netlify-cms

Reply:
✅ Done